Src/java/qdm/schema/qdm.5.3.xsd

```diff
@@ -272,6 +272,7 @@
 					<xs:element name="method" type="dt:Code" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="result" type="dt:Any" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="component" type="Component" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="relatedTo" type="dt:Code" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
We don't really have an "ID" type in the base CQL system types, so I kind of overloaded Code for that purpose. It's probably not entirely appropriate, but in the absence of a suitable ID type (specifically one that had a System), I think it's better to be able to use a Code, with the understanding that the only thing required is the "code" element that would contain the identifier as a string. The "system" element can then be used to provide a namespace if necessary.